igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
```

Comment:
I think this file used to be indented 4 spaces, and now it's mixed with 2 and 4 space indents. Could you change it back to 4?